plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsToolkitExplorerFactory.kt

```diff
@@ -80,7 +80,10 @@ class AwsToolkitExplorerFactory : ToolWindowFactory, DumbAware {
         val component = if (!isTookitConnected(project) || shouldPromptToolkitReauth(project)) {
             ToolkitWebviewPanel.getInstance(project).component
         } else {
-            AwsToolkitExplorerToolWindow.getInstance(project)
+            // TODO: it's a hack to prevent ToolkitBrowser.prepareUi not working before the browser is rendered on the UI
```

Comment:
Do we need the if-else block?

Reply:
no, we don't need it at this point of time